scripts/easyphoto_infer.py

```
@@ -517,28 +527,42 @@ def easyphoto_infer_forward(
-            resize          = float(short_side / 768.0)
-            new_size        = (int(output_image.width//resize), int(output_image.height//resize))
-            output_image    = output_image.resize(new_size, Image.Resampling.LANCZOS)
-            # When reconstructing the entire background, use smaller denoise values with larger diffusion_steps to prevent discordant scenes and image collapse.
-            output_image    = inpaint_only(output_image, output_mask, input_prompt_without_lora, diffusion_steps=20 if not background_restore else 30, denoising_strength=0.30 if not background_restore else 0.10, hr_scale=1, seed=str(seed), sd_model_checkpoint=sd_model_checkpoint)
-            
         try:
+            if min(len(template_face_safe_boxes), len(user_ids)) > 1 or background_restore:
+                logging.info("Start Thirt diffusion for background.")
+                output_image    = Image.fromarray(np.uint8(output_image))
+                short_side      = min(output_image.width, output_image.height)
+                if output_image.width / output_image.height > 1.5 or output_image.height / output_image.width > 1.5:
+                    target_short_side = 512
+                else:
+                    target_short_side = 768
+                resize          = float(short_side / target_short_side)
+                new_size        = (int(output_image.width//resize), int(output_image.height//resize))
+                output_image    = output_image.resize(new_size, Image.Resampling.LANCZOS)
+                # When reconstructing the entire background, use smaller denoise values with larger diffusion_steps to prevent discordant scenes and image collapse.
+                output_image    = inpaint_only(output_image, output_mask, input_prompt_without_lora, diffusion_steps=20 if not background_restore else 30, denoising_strength=0.30 if not background_restore else 0.10, hr_scale=1, seed=str(seed), sd_model_checkpoint=sd_model_checkpoint)
+        except:
```

donot use bare except, catch e and print out